include/vinecopulib/bicop/implementation/kernel.ipp

```diff
@@ -16,50 +17,80 @@ inline KernelBicop::KernelBicop()
   Eigen::VectorXd grid_points(m);
   for (size_t i = 0; i < m; ++i)
     grid_points(i) = -3.25 + i * (6.5 / static_cast<double>(m - 1));
-  grid_points = tools_stats::pnorm(grid_points);
-  
-  // move boundary points to 0/1, so we don't have to extrapolate
-  grid_points(0) = 0.0;
```

Comment:
Why don't we need that anymore?

Reply:
Oh good catch, I started the branch before https://github.com/vinecopulib/vinecopulib/commit/89696890395fc1083faaf01a6119c93a98106cec.